### PR TITLE
(feature) restrict access to admin/jobs and admin/feedback

### PR DESCRIPTION
**Pull request by despo**
to members with the admin role

As the admin/application_controller checks if a user is either an admin or an organiser for now add a super_admin/application_controller.

This could benefit from additional refactoring so there is better separation of what the organiser and admin roles can do. The tests should ideally also be refactored to reflect this.

Tackles issue #1163

### CODE REVIEW

File: app/controllers/admin/feedback_controller.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class Admin::FeedbackController < Admin::ApplicationController
+class Admin::FeedbackController < SuperAdmin::ApplicationController
```
Comment (matyikriszta) [inline app/controllers/admin/feedback_controller.rb]: @despo does this mean we don't want to give organizers access to feedbacks?

Comment (despo) [reply app/controllers/admin/feedback_controller.rb]: They can still access the feedback that has been given for their chapter specifically.

Comment (matyikriszta) [reply app/controllers/admin/feedback_controller.rb]: @despo ok, cool

File: app/controllers/coach/feedback_controller.rb [hunk 2]
```diff
@@ -1,14 +0,0 @@
-class Coach::FeedbackController < Coach::ApplicationController
```
Comment (matyikriszta) [inline app/controllers/coach/feedback_controller.rb]: @despo does this mean we also removed feedback access for coaches?

Comment (despo) [reply app/controllers/coach/feedback_controller.rb]: It does. I think it's something we can clarify and revisit if necessary as we still get names posted and we don't just render feedback that was specifically left for the coach viewing the page.

Comment (despo) [reply app/controllers/coach/feedback_controller.rb]: In addition to this the link was never explicitly exposed to coaches (it was part of the organiser menu) so not many people are/were aware and have ever access this.

Comment (matyikriszta) [reply app/controllers/coach/feedback_controller.rb]: @despo ok, sure

**Pull request merged**